test/hotspot/jtreg/compiler/intrinsics/sha/cli/TestUseSHA3IntrinsicsOptionOnSupportedCPU.java

```diff
@@ -28,6 +28,7 @@
  * @summary Verify UseSHA3Intrinsics option processing on supported CPU.
  * @library /test/lib /
  * @requires vm.flagless
+ * @requires os.arch == "aarch64" & os.family == "mac"
```

Comment:
I think this requires a comment. Something to the effect that we don't always enable the use of SHA3 instructions because they don't always help.

Reply:
Agree. Updated in the latest commit.
Would you mind taking another look? Thanks!